library/x509_crt.c

```diff
@@ -955,6 +956,10 @@ static int x509_get_crt_ext( unsigned char **p,
 
         if( ret != 0 )
         {
+            /* Give the callback (if any) a chance to handle the extension */
+            if (cb && cb(crt, &extn_oid, is_critical, p, end_ext_octet) == 0)
```

Comment:
Style: we put spaces on the wrong side of parentheses. We avoid implicit usage of integers or pointers as booleans.
```suggestion
            if( cb != NULL && cb( crt, &extn_oid, is_critical, p, end_ext_octet ) == 0 )
```